frontend/libs/studio-components/src/components/StudioTextResourcePicker/StudioTextResourcePicker.test.tsx

```diff
@@ -89,6 +89,16 @@ describe('StudioTextResourcePicker', () => {
     expect(onValueChange).toHaveBeenCalledWith(null);
   });
 
+  it('Does not apply other changes to the textfield than the ones triggered by the user when the user changes from a valid to an invalid value', async () => {
+    const user = userEvent.setup();
+    const chosenTextResource = textResources[129];
```

Comment:
What is the 129 number? 😄 
Can we refactor it out in a variable with a meaningful name?

Reply:
The number is random. It could be anything that is a valid index of the `textResources` array. I agree that this would become clearer with a designated variable, well noticed!

Reply:
I have fixed this now. I created a variable named `arbitraryTextResourceIndex` and replaced all instances of this number in the file with this variable.